common/ideas/00_basic_ideas.txt

```diff
@@ -632,7 +599,7 @@ exploration_ideas = {
 }
 
 
-maritime_ideas = {
+navigation_ideas = {
```

Comment:
I like what's in here, but I'm not sure whether it really has a very realistic use case. It improves neither naval strength or income by a really significant amount. It feels like a mix between trade and maritime ideas, while fulfilling the role of neither of them. It might needs either or both a direct increase to income (trade efficiency or goods produced) and naval quality (morale, engagement width, durability, combat ability etc). A country with this idea group should probably be able to have a somewhat stronger navy than a country without it, and probably also be able to somewhat compete with a country with naval ideas.

Reply:
Perhaps a significant buff to galleys could be interesting? It would provide an alternative to the buff to heavies that naval gives

Reply:
I do like the galley idea, plus we could change it so that galley width would be 0.5 like how it will be in 1.31.